### PR TITLE
Add pymarkovchain

### PR DESCRIPTION
**Pull request by asmeurer**

### CODE REVIEW

File: recipes/pymarkovchain/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8a03fba5d9f390b528991f5e3411188b545aa16a7131cea44c5f9cf1814d08a
+
+build:
```
Comment (synapticarbors) [inline recipes/pymarkovchain/meta.yaml]: Can we make this `noarch: python`?

File: recipes/pymarkovchain/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - pymarkovchain
+
+about:
+  home: https://github.com/MaxWagner/PyMarkovChain
+  license: "BEER-WARE"
```
Comment (synapticarbors) [inline recipes/pymarkovchain/meta.yaml]: I think the "BEER-WARE" license requires the `license_file`. I think it should be there in the source distribution, but I haven't checked.

Comment (asmeurer) [reply recipes/pymarkovchain/meta.yaml]: The license itself doesn't require much https://github.com/TehMillhouse/PyMarkovChain/blob/master/LICENSE.txt

Comment (synapticarbors) [reply recipes/pymarkovchain/meta.yaml]: I interpretted "As long as you retain this notice..." as meaning you should include the notice with any distribution of the code.

Comment (asmeurer) [reply recipes/pymarkovchain/meta.yaml]: I see. So conda-forge requires the license file for any license, except may be unlicense. This should be clarified in the example recipe.

**Comment by asmeurer** [comment]
I don't know what happened on Windows.

**Comment by asmeurer** [comment]
Is this a bug in the libarchive library that conda build uses? @msarahan 

```pytb
Downloading https://pypi.io/packages/source/P/PyMarkovChain/PyMarkovChain-1.8.tar.gz
Success
Extracting download
Traceback (most recent call last):
  File ".ci_support\build_all.py", line 83, in <module>
    build_all(args.recipes_dir, args.arch)
  File ".ci_support\build_all.py", line 73, in build_all
    conda_build.api.build(list(d.keys()), config=config)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\api.py", line 201, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\build.py", line 2255, in build_tree
    notest=notest,
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\build.py", line 1405, in build
    try_download(m, no_download_source=False, raise_error=True)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\render.py", line 580, in try_download
    source.provide(metadata)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\source.py", line 633, in provide
    timeout=metadata.config.timeout, locking=metadata.config.locking)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\source.py", line 143, in unpack
    tar_xf(src_path, tmpdir)
  File "C:\Miniconda36-x64\lib\site-packages\conda_build\utils.py", line 639, in tar_xf
    libarchive.extract_file(tarball, flags)
  File "C:\Miniconda36-x64\lib\site-packages\libarchive\extract.py", line 70, in extract_file
    extract_entries(archive, flags)
  File "C:\Miniconda36-x64\lib\site-packages\libarchive\extract.py", line 50, in extract_entries
    write_header(write_p, entry._entry_p)
  File "C:\Miniconda36-x64\lib\site-packages\libarchive\ffi.py", line 91, in check_int
    raise archive_error(args[0], retcode)
  File "C:\Miniconda36-x64\lib\site-packages\libarchive\ffi.py", line 75, in archive_error
    raise ArchiveError(msg, errno(archive_p), retcode, archive_p)
libarchive.exception.ArchiveError: Can't create '\\?\C:\bld\tmpp9fvkpv2\PyMarkovChain-1.8\README' (errno=1, retcode=-25, archive_p=160352691488)
```

And are there any workarounds?

**Comment by asmeurer** [comment]
That's OK I don't need this anymore.

**Pull request closed without merging**